packages/main/src/MultiInput.js

```diff
@@ -182,18 +182,17 @@ class MultiInput extends Input {
 
 		const isHomeInBeginning = isHome(event) && event.target.selectionStart === 0;
 
-		if (isLeft(event) || isHomeInBeginning) {
-			this._skipOpenSuggestions = true; // Prevent input focus when navigating through the tokens.
+		if (isHomeInBeginning) {
+			this._skipOpenSuggestions = true; // Prevent input focus when navigating through the tokens
+			return this._focusFirstToken(event);
+		}
 
+		if (isLeft(event) || isBackSpace(event)) {
```

Comment:
Here, I will write all the things regarding the BACKSPACE handling:
When we are not attached to "token-delete" event and a token could not be deleted on BACKSPACE, it looks a little bit strange for me - when the focus is moved from one to the previous token on BACKSPACE the caret is flickering in the input field. Going to the first token, on next BACKSPACE the focus is moved again the second one and then on follow interactions via BACKSPACE it is changed between the first and the second token.
Moreover, when we have not editable tokens, now in the ui5-multiinput on BACKSPACE over such token, the focus remains over it and does not go the the next token. In openUI5, if you have a readonly token, when the focus is over it, on BACKSPACE the focus is moved to the previous one. That should be discussed with the team because it is inconsistent.

Reply:
This is now fixed, however in order not to move focus when a token deletion is not handled, we should check if the token exists inside the setTimeout callback, as before that the DOM is not repainted. This means that the focusing of the next token (when the current one **actually** gets deleted) will happen with a slight delay. Is this approach feasible or are we fine with the focus jumping remaining as it is?

```js
setTimeout(() => {
	const deletedTokenIdx = token ? this._tokens.indexOf(token) : this._tokens.indexOf(event.target);
	if (nextToken && deletedTokenIdx === -1) { // only if deleted, move focus and set current item to be nextToken
		this._itemNav.setCurrentItem(nextToken);
		nextToken.focus();
	}
}, 0);
```

Reply:
Agreed on removing this part of the fix, as it's out of scope.